### PR TITLE
Reduce the size of module

### PR DESCRIPTION
**Pull request by polemius**
@ai I've split the module on to files. And we reduce the 25 B. Did you mean something like this?

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,102 +3,104 @@
...
-        }
-
-        return next
-      })
-    },
-    UNDO: undo,
-    REDO: redo
+module.exports = {
+  createHistory: function (paths) {
+    paths = paths || []
```
Comment (ai) [inline index.js]: You can remove it and ask users always pass an array. It will be not a problem for most of the cases and will reduce the size.

**Comment by ai** [comment]
Yeap :+1:

File: index.js (hunk 1 above)
Comment (polemius) [reply index.js]: How I can ask the users? By definition in JSDoc? or in README?

Comment (ai) [reply index.js]: By `README.md`. ANyway they will copy-paste the example.

If you want better DX, you can do:

```js
if (process.env.NODE_ENV === 'development') {
  if (!paths) {
    throw new Error('better error text')
  }
}
```

Comment (polemius) [reply index.js]: Ok, I've removed the `paths = paths || []` and add the error as you proposed. Please confirmed and I will merge this PR and release a new version.

**Comment by ai** [comment]
LGTM

**Comment by polemius** [comment]
Fixed #9

**Pull request merged**